tests/terraform/checks/resource/aws/test_RedShiftSSL.py

```diff
@@ -1,66 +1,41 @@
-                  }
-                  
-                  parameter {
-                    name  = "enable_user_activity_logging"
-                    value = "true"
-                  }
-                }
-
-        """)
-        resource_conf = hcl_res['resource'][0]['aws_redshift_parameter_group']['examplea']
-        scan_result = check.scan_resource_conf(conf=resource_conf)
-        self.assertEqual(CheckResult.FAILED, scan_result)
-
-    def test_failure_no_parameters(self):
-        hcl_res = hcl2.loads("""
-                resource "aws_redshift_parameter_group" "examplea" {
-                  name   = var.param_group_name
-                  family = "redshift-1.0"
-                  
-                }
-       """)
-        resource_conf = hcl_res['resource'][0]['aws_redshift_parameter_group']['examplea']
-        scan_result = check.scan_resource_conf(conf=resource_conf)
-        self.assertEqual(CheckResult.FAILED, scan_result)
-
-    def test_success_with_parameters(self):
-        hcl_res = hcl2.loads("""
-                resource "aws_redshift_parameter_group" "examplea" {
-                  name   = var.param_group_name
-                  family = "redshift-1.0"
-                  
-                  parameter {
-                    name  = "require_ssl"
-                    value = "true"
-                  }
-                  
-                  parameter {
-                    name  = "enable_user_activity_logging"
-                    value = "true"
-                  }
-                }
-        """)
-        resource_conf = hcl_res['resource'][0]['aws_redshift_parameter_group']['examplea']
-        scan_result = check.scan_resource_conf(conf=resource_conf)
-        self.assertEqual(CheckResult.PASSED, scan_result)
-
-if __name__ == '__main__':
+    def test(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_RedShiftSSL"
+        report = runner.run(
+            root_folder=test_files_dir, runner_filter=RunnerFilter(checks=[check.id])
+        )
+        summary = report.get_summary()
+
+        passing_resources = {
+            "aws_redshift_parameter_group.pass",
+            "aws_redshift_parameter_group.passbutbool",  
+        }
+        failing_resources = {
+            "aws_redshift_parameter_group.fail",
+            "aws_redshift_parameter_group.failasfalse"
+        }
+
+        passed_check_resources = set([c.resource for c in report.passed_checks])
+        failed_check_resources = set([c.resource for c in report.failed_checks])
+
+        self.assertEqual(summary["passed"], 2)
+        self.assertEqual(summary["failed"], 2)
+        self.assertEqual(summary["skipped"], 0)
+        self.assertEqual(summary["parsing_errors"], 0)
+
+        self.assertEqual(passing_resources, passed_check_resources)
+        self.assertEqual(failing_resources, failed_check_resources)
+
+
+if __name__ == "__main__":
```

Comment:
@JamesWoolfenden Can you remove this?

Reply:
which bit?

Reply:
@nimrodkor ?

Reply:
This line:
`if __name__ == "__main__":`

Reply:
i think that line is required. It looks like I added it but its just the layout of the difftool, i think?